sql/hive/src/main/scala/org/apache/spark/sql/hive/test/TestHiveContext.scala

```diff
@@ -37,39 +38,36 @@ import org.apache.spark.sql.execution.CacheTableCommand
 import org.apache.spark.sql.hive._
 import org.apache.spark.sql.hive.execution.HiveNativeCommand
 import org.apache.spark.util.Utils
-import org.apache.spark.{SparkConf, SparkContext}
 
 /* Implicit conversions */
 import scala.collection.JavaConversions._
 
-// SPARK-3729: Test key required to check for initialization errors with config.
-object TestHive
```

Comment:
do you mean outside of Spark? The point is to never use this in Spark tests, since even referencing it will destabilize the tests because it leaves two overlapping `SparkContext`s in the same JVM